docs/source/contents/4_draft_mapping.rst

```diff
@@ -1825,7 +1825,7 @@ Here's an `example record - ruskin:108 <https://digital.lib.utk.edu/collections/
 
 .. code-block:: xml
 
-    <form authority="http://vocab.getty.edu/aat/300046300">photographs</form>
+    <form authority="aat" valueURI="http://vocab.getty.edu/aat/300046300">photographs</form>
```

Comment:
Addresses https://hyp.is/7chIZhgbEe2Ni2f2-FGM7w/utk-mods-to-rdf.readthedocs.io/en/latest/contents/4_draft_mapping.html